ddtrace/constants.py

```diff
@@ -1,23 +1,66 @@
-FILTERS_KEY = "FILTERS"
-SAMPLE_RATE_METRIC_KEY = "_sample_rate"
-SAMPLING_PRIORITY_KEY = "_sampling_priority_v1"
-ANALYTICS_SAMPLE_RATE_KEY = "_dd1.sr.eausr"
-SAMPLING_AGENT_DECISION = "_dd.agent_psr"
-SAMPLING_RULE_DECISION = "_dd.rule_psr"
-SAMPLING_LIMIT_DECISION = "_dd.limit_psr"
-ORIGIN_KEY = "_dd.origin"
-HOSTNAME_KEY = "_dd.hostname"
-ENV_KEY = "env"
-VERSION_KEY = "version"
-SERVICE_KEY = "service.name"
-SERVICE_VERSION_KEY = "service.version"
-SPAN_KIND = "span.kind"
-SPAN_MEASURED_KEY = "_dd.measured"
-KEEP_SPANS_RATE_KEY = "_dd.tracer_kr"
-
-NUMERIC_TAGS = (ANALYTICS_SAMPLE_RATE_KEY,)
-
-MANUAL_DROP_KEY = "manual.drop"
-MANUAL_KEEP_KEY = "manual.keep"
-
-LOG_SPAN_KEY = "__datadog_log_span"
+import sys
+
+from ddtrace.internal.constants import ANALYTICS_SAMPLE_RATE_KEY
+from ddtrace.internal.constants import MANUAL_DROP_KEY
+from ddtrace.internal.constants import MANUAL_KEEP_KEY
+from ddtrace.internal.constants import ORIGIN_KEY
+from ddtrace.internal.constants import SAMPLING_PRIORITY_KEY
+from ddtrace.internal.constants import SERVICE_KEY
+from ddtrace.internal.constants import SERVICE_VERSION_KEY
+from ddtrace.internal.constants import SPAN_KIND
+from ddtrace.internal.constants import SPAN_MEASURED_KEY
+from ddtrace.internal.constants import VERSION_KEY
+from ddtrace.vendor import debtcollector
+
+
+__all__ = [
+    "ANALYTICS_SAMPLE_RATE_KEY",
+    "MANUAL_DROP_KEY",
+    "MANUAL_KEEP_KEY",
+    "ORIGIN_KEY",
+    "SAMPLING_PRIORITY_KEY",
+    "SERVICE_KEY",
+    "SERVICE_VERSION_KEY",
+    "SPAN_KIND",
+    "SPAN_MEASURED_KEY",
+    "VERSION_KEY",
+]
+
+
+if sys.version_info >= (3, 7, 0):
```

Comment:
Can we add https://github.com/facelessuser/pep562?

Reply:
@majorgreys done